src/Command/CacheFlushAllCommand.php

```diff
@@ -50,6 +51,9 @@ public function execute(InputInterface $input, OutputInterface $output): void
             }
         }
 
+        // The current event dispatcher is stale, let's not use it anymore
+        $this->getApplication()->setDispatcher(new EventDispatcher());
```

Comment:
Is this still needed in the `NEXT_MAJOR`?

Reply:
IMO yes @OskarStark it is, even in next major, to prevent further execution of event without symfony cache